File: AdvancedLevel_C++/1035. Password (20).cpp
#include <iostream>
#include <vector>
using namespace std;
int main() {
    int n;
    scanf("%d", &n);
    vector<string> v;
    for(int i = 0; i < n; i++) {
        string name, s;
        cin >> name >> s;
        int len = s.length(), flag = 0;
        for(int j = 0; j < len; j++) {
            switch(s[j]) {
                case '1' : s[j] = '@'; flag = 1; break;
                case '0' : s[j] = '%'; flag = 1; break;
                case 'l' : s[j] = 'L'; flag = 1; break;
                case 'O' : s[j] = 'o'; flag = 1; break;
            }
        }
        if(flag) {
            string temp = name + " " + s;
            v.push_back(temp);
        }
    }
    int cnt = v.size();
    if(cnt != 0) {
        printf("%d\n", cnt);
        for(int i = 0; i < cnt; i++)
            cout << v[i] << endl;
    } else if(n == 1) {
        printf("There is 1 account and no account is modified");
    } else {
        printf("There are %d accounts and no account is modified", n);
    }
    return 0;
}

File: AdvancedLevel_Java/1035 Password (20).java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.ArrayList;


public class Main {
    static class Node {
        String username;
        char[] password;

        public Node(String username, char[] password) {
            this.username = username;
            this.password = password;
        }
    }

    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int n = Integer.parseInt(br.readLine());
        ArrayList<Node> list = new ArrayList<Node>();
        
        for (int i = 0; i < n; i++) {
            String[] line = br.readLine().split(" ");
            char[] chs = line[1].toCharArray();
            
            boolean flag = false;
            for (int j = 0; j < chs.length; j++) {
                if (chs[j] == '1') {
                    chs[j] = '@';
                    flag = true;
                } else if (chs[j] == '0') {
                    chs[j] = '%';
                    flag = true;
                } else if (chs[j] == 'l') {
                    chs[j] = 'L';
                    flag = true;
                } else if (chs[j] == 'O') {
                    chs[j] = 'o';
                    flag = true;
                }
            }
            if (flag) {
                Node node = new Node(line[0], chs);
                list.add(node);
            }
        }
        if (list.size() == 0) {
            if (n == 1) {
                System.out.println("There is " + n + " account and no account is modified");
            } else {
                System.out.println("There are " + n + " accounts and no account is modified");
            }
        } else {
            System.out.println(list.size());
            for (Node item : list) {
                System.out.print(item.username + " ");
                System.out.println(item.password);
            }
        }

    }

}
